test/gtest/ucp/test_ucp_am.cc

```diff
@@ -863,6 +863,55 @@ UCS_TEST_P(test_ucp_am_nbx_send_flag, eager, "RNDV_THRESH=128")
 UCP_INSTANTIATE_TEST_CASE(test_ucp_am_nbx_send_flag)
 
 
+class test_ucp_am_nbx_short_slow_path : public test_ucp_am_nbx {
+public:
+    static void get_test_variants_reply(std::vector<ucp_test_variant> &variants)
+    {
+        add_variant_with_value(variants, UCP_FEATURE_AM, 0, "");
+        add_variant_with_value(variants, UCP_FEATURE_AM, UCP_AM_SEND_FLAG_REPLY,
+                               "reply");
+    }
+
+    static void get_test_variants(std::vector<ucp_test_variant> &variants)
+    {
+        add_variant_values(variants, get_test_variants_reply, 0);
+        add_variant_values(variants, get_test_variants_reply, 1, "proto");
+    }
+
+protected:
+    virtual unsigned get_send_flag() const
+    {
+        return get_variant_value(0);
+    }
+};
+
+UCS_TEST_P(test_ucp_am_nbx_short_slow_path, short_slow_path)
+{
+    size_t header_size = 8;
+    size_t data_size   = 8;
+    mem_buffer sbuf(data_size, tx_memtype());
+    sbuf.pattern_fill(SEED);
+    m_hdr.resize(header_size);
+    ucs::fill_random(m_hdr);
+    m_am_received = false;
+    ucs_status_ptr_t sptr;
```

Comment:
better declare and initialize variables right before they are used
for example:
`ucs_status_ptr_t sptr = ...`